api/app/db/crud/hfi_calc.py

```diff
@@ -78,3 +90,8 @@ def get_fire_centre_fire_start_ranges(session: Session, fire_centre_id: id) -> C
 def get_fire_start_lookup(session: Session) -> CursorResult:
     """ Get the fire start lookup table """
     return session.query(FireStartLookup)
+
+
+def get_fuel_type(session: Session, fuel_type_id: int) -> FuelType:
```

Comment:
could we slightly rename this function to `get_fuel_type_by_id`?

Reply:
I don't know if it's considered a best practice or not, but I just like it when the function name makes it clear what attribute the lookup is based on.